engine/client/client.go

```diff
@@ -155,7 +150,13 @@ func Connect(ctx context.Context, params Params) (_ *Client, _ context.Context,
 		c.SecretToken = os.Getenv("DAGGER_SESSION_TOKEN")
 		c.httpClient = &http.Client{
 			Transport: &http.Transport{
-				DialContext:       c.DialContext,
+				DialContext: c.DialContext,
+				// TODO: is disabling keepalives in this code path necessary?
+				// TODO: is disabling keepalives in this code path necessary?
+				// TODO: is disabling keepalives in this code path necessary?
+				// TODO: is disabling keepalives in this code path necessary?
+				// TODO: is disabling keepalives in this code path necessary?
+				// TODO: is disabling keepalives in this code path necessary?
```

Comment:
Yeah I am gonna clean those up; this one in particular I'll just remove I think. I suspect it's theoretically true that keepalives aren't needed when connecting direct over HTTP rather than doing grpc tunnelling (which is thankfully no longer needed for modules), but it's such a subtle change that could cause obscure race conditions that it's better to save for a different time I think.